client/src/pages/Profiles/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography, Box, Card } from '@mui/material';
+import useStyles from './useStyles';
+
+export default function ProfileCard({ profile }: any) {
+  const classes = useStyles();
+  const rates = ['$15', '$20', '$25', '$30'];
+  return (
+    <>
+      <Box
```

Comment:
Let's also make sure we use the Card API as each card contains information about a particular sitter

Ref: https://mui.com/api/card/ (Please refer to some examples)